source/AndroidXProject.cshtml

```diff
@@ -123,17 +127,23 @@
 
   @if (@Model.MavenArtifacts.Count > 0) {
   <ItemGroup>
-    @foreach (var art in @Model.MavenArtifacts) {
-      if (art.MavenArtifactPackaging == "aar") {
+    @foreach (var art in @Model.MavenArtifacts) 
+    {
+      if (art.MavenArtifactPackaging == "aar") 
+      {
     <InputJar Include="..\..\externals\@(art.MavenGroupId)\@(art.MavenArtifactId)\classes.jar" />
     <!-- For those artifacts with lib/ folder -->
     <InputJar 
       Condition="Exists('..\..\externals\@(art.MavenGroupId)\@(art.MavenArtifactId)\libs\')"
       Include="..\..\externals\@(art.MavenGroupId)\@(art.MavenArtifactId)\libs\*.jar" 
       />
-      } else {
+      } 
+      else 
+      {
     <InputJar Include="..\..\externals\@(art.MavenGroupId)\@(art.MavenArtifactId).jar" />
       }
+    <InputJar Remove="..\..\externals\@(art.MavenGroupId)\@(art.MavenArtifactId)\lint.jar" />
```

Comment:
These are getting removed in the cake script... So is it that you're excluding them in the nupkg still?

Reply:
Yes, but removal from the cake script did not work (and still does not work), because we are unzipping everything in obj/ folder, so removal from aar in the cake script was the only solution. This was remains from preventing `lint.jar` from being included before it was removed from aar.

AndroidX issue:

https://github.com/xamarin/AndroidX/issues/73

Android PR for Msbuild tasks with more details:

https://github.com/xamarin/xamarin-android/pull/4385

Reply:
removed lint.jar from template
Now removal only in cake script until fix is available.